openslides/users/static/templates/users/user-detail.html

```diff
@@ -33,7 +33,7 @@ <h2 translate>Participant</h2>
       <label translate>Participant number</label>
         {{ user.number }}
       <label translate>About me</label>
-        <div ng-bind-html="user.about_me"></div>
+        <div ng-bind-html="user.about_me | trusted"></div>
```

Comment:
Uuuh. Do you really want users to add whatever they want? Everybody could get an exploit here!?